.github/workflows/deploy-to-demoee.yml

```
@@ -0,0 +1,19 @@
+name: Deploy to demoee.org
+
+on:
+    push:
+        branches: [master]
```

To test the things, you can temporarily do this.
```suggestion
        branches: [master, deploy-to-demoee-org]
```